guarddog/analyzer/sourcecode/exfiltrate-sensitive-data.yml

```diff
@@ -31,6 +31,18 @@ rules:
           - metavariable-regex:
               metavariable: $ENVVAR
               regex: ([\"\'](AWS_ACCESS_KEY_ID|AWS_SECRET_ACCESS_KEY|AWS_SESSION_TOKEN)[\"\'])
+      - patterns:
+          - pattern-inside: |
+              $CONNECT = sqlite3.connect(...)
+              ...
+              $CURSOR = $CONNECT.cursor(...)
+              ...
+          - pattern: $CURSOR.execute($QUERY, ...)
+          - metavariable-pattern:
+              metavariable: $QUERY
+              patterns:
+                - pattern: "..."
+                - pattern-regex: (?i)select\s+\S+.*\s+from\s+(cookies|credit_cards|logins|moz_cookies|moz_formhistory|moz_logins)
```

Comment:
you are still having the `select \s+...` here
lets stick only to the table names

Reply:
Oh, gotcha.  Do you think we should we leave the `from\s+` to identify them as table names or is it okay without?

Reply:
idk how likely it is to find false positives without the `from\s` i would recommend removing it from the regex.
we can test it out by running against the top 1k packages